app/scripts/lib/setupAccountLabelsPetnamesBridge.test.ts

```diff
@@ -0,0 +1,153 @@
+import { NameController } from '@metamask/name-controller';
+import setupAccountLabelsPetnamesBridge, {
+  PertinentPreferencesController,
+  PertinentState,
+  ACCOUNT_LABEL_NAME_TYPE,
+  ACCOUNT_LABEL_CHAIN_ID,
+} from './setupAccountLabelsPetnamesBridge';
+
+const ADDRESS_MOCK = '0xabc';
+const NAME_MOCK = 'testName';
+const NAME_2_MOCK = 'testName2';
+
+interface MockPreferencesController
```

Comment:
Minor, but I find the simplest way to type the mocks in the unit tests is to set the return type to `jest.Mocked<SomeType>` and then just set the minimal requited properties and cast to `any`, as you've done with `createNameControllerMock`.

Reply:
I see what you mean. I think usually we'd just create a dedicated variable so we're only dealing with mocking functions at one level.

So in the `beforeEach`, we could have:

`preferencesControllerStoreMock = jest.mocked(preferencesControllerDefault.store);`